docs/contribute.md

```diff
@@ -1,328 +1,141 @@
 
-Once the project has been bootstrapped, you may want to edit generated `.env`
-file to set up available backends parameters that will be injected in the
-running container as environment variables to configure Ralph (see [backends
-documentation](./backends.md)):
+### 1. Search for Existing Issues
 
-```dotenv
-# Elasticsearch backend
-RALPH_ES_HOSTS=http://elasticsearch:9200
-RALPH_ES_INDEX=statements
-RALPH_ES_TEST_HOSTS=http://elasticsearch:9200
-RALPH_ES_TEST_INDEX=test-index
+Before creating a new issue, search the existing issues to see if your concern has already been raised. If you find a related issue, you can add your input or follow the discussion.
 
-# [...]
-```
+### 2. Creating a New Issue
 
-> Note that lines starting with a `#` are considered as commented and thus will
-> have no effect while running Ralph.
+**Bug Report**
 
-Now you can start playing the CLI:
+- Use this template if you've encountered a bug in our project.
+- Include a clear and concise title that summarizes the issue.
+- Provide detailed steps to reproduce the bug.
+- Include information about your environment (e.g., operating system, browser version, etc.).
 
-```
-$ bin/ralph --help
-```
+**Feature Request**
 
-To lint your code, either use the `lint` meta target or one of the linting tools we use:
+- Use this template if you have a feature you'd like to suggest.
+- Provide a descriptive title.
+- Explain the feature you're proposing and its potential benefits.
 
-```bash
-# Run all linters
-$ make lint
+**Support Question**
 
-# Run pylint
-$ make lint-pylint
+- Use this template if you have a question or need assistance with our project.
+- Provide a clear and concise title.
+-  Detail your question or problem.
 
-# List available linters
-$ make help | grep lint-
-```
+**Security Vulnerability**
 
-To run tests on your code, either use the `test` Make target or the
-`bin/pytest` script to pass specific arguments to the test runner:
+If you discover a security vulnerability, please report it directly to us following our Security Vulnerability Reporting Guidelines.
 
-```bash
-# Run all tests
-$ make test
+### 3. Code of conduct in discussion
 
-# Run pytest with options
-$ bin/pytest -x -k mixins
+- Be respectful and considerate when participating in discussions.
+- Avoid using offensive language, and maintain a positive and collaborative tone.
+- Stay on topic and avoid derailing discussions.
 
-# Run pytest with options and more debugging logs
-$ bin/pytest -x -vvv -s --log-level=DEBUG -k mixins
-```
+### 4. Collaboration
 
-### Working with local backends
+Feel free to engage in discussions, offer help, or provide feedback on existing issues. Your input is valuable in shaping the project's future.
 
-Not all backends are accessible in a local environment development; for now
-only `elasticsearch` and `swift` services are accessible as docker containers
-(see `docker-compose.yml` services).
+### 5. Closing issues
```

Comment:
I think this could be merged with "Creating a new issue" (maybe "Creating and closing issues" ?)

Alternatively, I would move this to be just after "Creating a new issue".